migtests/tests/import-file/run-import-file-test

```diff
@@ -55,17 +55,17 @@ main() {
 	# Test for fileOpts Flags having quote_char as single quote
 	step "Import data file: t1_quote_char.csv -> t1_quote_char"
 	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter '|' \
-			--file-table-map "t1_quote_char.csv:t1_quote_char"
+			--file-table-map "t1_quote_char.csv:t1_quote_char" --file-opts "quote_char='"
```

Comment:
Can you enhance the `t1_quote_char.csv` file to have a string containing single quote. I expect the default escape character (`"`) to be used to specify a single quote inside a string.

Reply:
Done